README.md

```diff
@@ -1,2 +1,124 @@
-# Queryalizer
-Query serializer, creating a query string from query object.
+# @repositive/query-parser
+
+[![License: LGPL v3](https://img.shields.io/badge/License-LGPL%20v3-blue.svg)](https://choosealicense.com/licenses/lgpl-3.0/)  
+[![npm version](https://badge.fury.io/js/%40repositive%2Fquery-parser.svg)](https://badge.fury.io/js/%40repositive%2Fquery-parser)  
+[![CircleCI](https://circleci.com/gh/repositive/query-parser.svg?style=svg)](https://circleci.com/gh/repositive/query-parser)  
+
+* [Installation](#installation)
+* [Available Serializers](#available-serializers)
+* [Usage](#usage)
+
+The purpose of this library is to transform search string to a intermediary tokenized data structure that can be used to perform analysis on the search or middle ground to translate the query into other DSL or query structures.
```

Comment:
The purpose of this library is to transform a search string to a ~~intermediary~~ tokenized data structure that can be used to perform analysis on the search input or as an intermediate structure to translate the query into other DSLs or query structures.